python/mxnet/rnn/rnn_cell.py

```diff
@@ -355,14 +355,20 @@ class LSTMCell(BaseRNNCell):
     params : RNNParams or None
         container for weight sharing between cells.
         created if None.
+    forget_bias : bias added to forget gate, default 0.0.
+        Jozefowicz et al. 2015 recommends setting this to 1.0
+        but the default is set to 0.0 for backward compatibility.
+
     """
-    def __init__(self, num_hidden, prefix='lstm_', params=None):
+    def __init__(self, num_hidden, prefix='lstm_', params=None, forget_bias=1.0):
```

Comment:
Instead of adding new parameter to the constructors and introducing new default behavior, how about the following:
- extend RNNParams (e.g. LSTMParams)
- Add the initialization behavior to the constructor of that new class
- When using this, user invokes the constructors with an instance of the new Params class.